### PR TITLE
Feature/notifications

### PR DESCRIPTION
**Pull request by matthewcarlreetz**

### CODE REVIEW

File: src/features/notifications/NoNotifications.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import Text from '../../components/Text'
+import BellBubble from '../../assets/images/bellBubble.svg'
+import Box from '../../components/Box'
+
+const NoNotifications = () => {
```
Comment (allenan) [inline src/features/notifications/NoNotifications.tsx:7]: could maybe call this `EmptyNotifications` or something? The `NoNo` in `NoNotifications` kind of runs together when I skim it 😅

File: src/utils/secureAccount.ts [hunk 2]
```diff
@@ -82,5 +83,46 @@ export const getKeypair = async (): Promise<Keypair | undefined> => {
   }
 }
 
+const sign = async (message: string) => {
+  // TODO: is there a better way to sign?
```
Comment (allenan) [inline src/utils/secureAccount.ts:87]: can we use the `Keypair#sign` function?

File: src/features/notifications/NoNotifications.tsx (hunk 1 above)
Comment (matthewcarlreetz) [reply src/features/notifications/NoNotifications.tsx:7]: ok, I'll rename it in my next PR, if that's all right.

File: src/store/account/accountSlice.ts [hunk 3]
```diff
@@ -6,30 +6,58 @@ import {
...
   'account/fetchData',
   async () => {
-    try {
-      const data = await Promise.all([getHotspots(), getAccount()])
-      return { hotspots: data[0] || [], account: data[1] }
-    } catch (e) {
-      console.log(e)
-      return { hotspots: [] }
+    const data = await Promise.all(
+      [getHotspots(), getAccount(), getWallet('notifications')].map((p) =>
```
Comment (tyler-whitman) [inline src/store/account/accountSlice.ts:51]: So does this just load once on app start?

File: src/features/notifications/NoNotifications.tsx (hunk 1 above)
Comment (allenan) [reply src/features/notifications/NoNotifications.tsx:7]: np

File: src/store/account/accountSlice.ts (hunk 3 above)
Comment (matthewcarlreetz) [reply src/store/account/accountSlice.ts:51]: for the moment, yes, but we will need to figure out a strategy around refreshing it.

**Review by tyler-whitman — APPROVED** [review]

**Pull request merged**